tests/test_english_empty.txt

```diff
@@ -0,0 +1,4 @@
+
+While this email is undeliverable, this test case makes sure that the
+parser does not crash and that most fields are nullable and the rest
+has sane defaults (e.g. text/plain Content Type).
```

Comment:
[SUGGESTION] What do you think about slightly increasing readability by adding commas? I'm thinking about something like that:

`While this email is undeliverable, this test case makes sure that the
parser does not crash, most fields are nullable, and the rest
has sane defaults (e.g. text/plain Content Type).`

Reply:
Fixed in https://github.com/mnako/letters/pull/13/commits/e531bb797130c121e7f62e468b362a02a9c43d1b, thank you!